sdk-python/omnilogger/database.py

```diff
@@ -10,7 +11,7 @@ def send_to_db(url, log):
 
     cursor.execute(
         """
-    INSERT INTO llm_logs (datetime_utc, input_string, output_string, total_tokens)
+    INSERT INTO llm_logs (datetime_utc, input_string, output_string,total_tokens)
```

Comment:
Removing this space makes the line ugly....

Reply:
good catch! i didnt realise that happened haha